src/mobile_responsive.js

```
@@ -0,0 +1,783 @@
+  setTimeout( () => {
+  //(Re)-enable content Editable ( If it was disabled when "OnFocus" was called without click)
+  if ( lastClickEl.closest('.contenteditableDisabled') !== null  )
+  {
+    var editableEl=lastClickEl.closest('.contenteditableDisabled');
+    lastClickEl.closest('.contenteditableDisabled').setAttribute('contenteditable', true);
+    lastClickEl.closest('.contenteditableDisabled').classList.remove('contenteditableDisabled') 
+    editableEl.focus();
+  }
+  if ( lastClickEl.querySelector('.contenteditableDisabled') !== null  )
+  {
+    var editableEl=lastClickEl.querySelector('.contenteditableDisabled');
```

Biome (2.1.2)

[error] Shouldn't redeclare 'editableEl'. Consider to delete it or rename it.

'editableEl' is defined here:


(lint/suspicious/noRedeclare)